lib/parser.mly

```diff
@@ -482,10 +489,12 @@ lhs:
 statement:
   | s=atomic_statement
     {  grammar_logger "atomic_statement" ;
-       {stmt_untyped= s; stmt_untyped_loc= Location ($startpos, $endpos)} }
+       {stmt_untyped= s;
+        stmt_untyped_loc=location_span_of_pos $startpos $endpos} }
   | s=nested_statement
     {  grammar_logger "nested_statement" ;
-       {stmt_untyped= s; stmt_untyped_loc= Location ($startpos, $endpos)} }
+       {stmt_untyped= s;
```

Comment:
[comment]
I like this reviewing that's just the same replacement over and over.  It's soothing :-)

Reply:
:-)